app/views/contexts/summary/answer.py

```diff
@@ -17,4 +29,16 @@ def serialize(self):
             "unit": self.unit,
             "unit_length": self.unit_length,
             "currency": self.currency,
+            "link": self.link,
         }
+
+    def _build_link(self, block_id, list_name, list_item_id, return_to):
```

Comment:
Minor but I think we should make these keyword only args, then it would be easy to follow and less chance of messing up the order.